docs/deployment-process/operations-runbooks/index.md

```diff
@@ -57,13 +57,34 @@ You can **Run** a **Snapshot** as many times as you want to.
 
 ## Environments Selection
 
-We don't believe that channels or lifecycles (progression) make sense for runbooks. Runbooks can be run on any environments you have access to. The interface has been designed to let you run a runbook quickly, so there's a single `Run on...` screen where you choose environments and specify any `Advanced` options, and then you run it.
+We don't believe that channels or lifecycles (progression) make sense for runbooks. Runbooks can be run on any environments you have access to. The interface has been designed to let you run a runbook quickly, so there's a single `Run...` screen where you choose environments and specify any `Advanced` options, and then you run it.
 
-## Permissions
+## Permissions and Roles
 
-Two permissions are available for the top-level Runbooks document: `RunbookView` (for viewing) and `RunbookEdit` (for creating, modifying, and deleting), so if you want to lock down the ability for your teams to create runbooks, you just need to disable these permissions.
+Permissions are available to help you manage access to Runbooks, these include `RunbookView` (for viewing), `RunbookEdit` (for creating, editing, deleting and snapshotting), and various `RunbookRun*` permissions.
```

Comment:
> and various `RunbookRun*` 
maybe just list the 3?

could mention it follows the pattern fo the Deployment permissions